chatbot/app/secret_key.py

```diff
@@ -1,6 +1,6 @@
 #Go to the OpenAI website and create an account there also,
 #make your own API key , which the OpenAI will give you for free. 
-#Place that KEY between the empty strings
```

Comment:
why have you changed this line

Reply:
Because it was my own personal API Of the OpenAi project , so the person who wants to use it will place his own APi Key , which the OpenAi will give him while signup .